db/migrate/20181001131632_add_conversion_host_id_to_miq_request_tasks.rb

```diff
@@ -0,0 +1,48 @@
+class AddConversionHostIdToMiqRequestTasks < ActiveRecord::Migration[5.0]
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options, Hash
+    belongs_to :conversion_host, :class_name => "AddConversionHostIdToMiqRequestTasks::ConversionHost"
+  end
+
+  class ServiceTemplateTransformationPlanTask < MiqRequestTask
+  end
+
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :service_template_transformation_plan_tasks, :class_name => "AddConversionHostIdToMiqRequestTasks::MiqRequestTask"
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :tags, :class_name => "AddConversionHostIdToMiqRequestTasks::Tag"
+  end
+
+  def up
+    add_column :miq_request_tasks, :conversion_host_id, :bigint
+
+    ServiceTemplateTransformationPlanTask.all.reject { |task| task.options[:transformation_host_id].nil? }.each do |task|
```

Comment:
A few things here ...
First, I don't think this query will work the way you're intending it to. STI is disabled so I think you would be better off using the base class and specifying the `type` in a `where` clause.

Also, instead of using `reject` here, can we make this a guard clause in the `each` block? That will simplify this and prevent us from iterating through the list of tasks twice.

So something like this:
```ruby
MiqRequestTask.where(:type => "ServiceTemplateTransformationPlanTask").each do |task|
  next unless task.options[:transformation_host_id]
  ...
end
```